ivy_tests/test_ivy/helpers/hypothesis_helpers/dtype_helpers.py

```diff
@@ -99,11 +99,53 @@ def get_dtypes(
         returns the complete list of valid types
     none
         allow none in the list of valid types
+    key
+        TODO: add description
+    prune_function
+        TODO: add description
```

Comment:
if `prune_function` is True, the drawn data types will be specific for the currently running test.